package.json

```diff
@@ -58,16 +58,16 @@
     "gulp": "^4.0.2",
     "gulp-rename": "^1.4.0",
     "gulp-tslint": "^8.1.3",
-    "mocha": "^6.2.1",
+    "minimist": ">=1.2.5",
+    "mocha": "^8.1.1",
```

Comment:
This made me realize, we don't even use mocha in vscode-xml. We can remove this and the typescript definitions here: 

https://github.com/redhat-developer/vscode-xml/blob/24daed620f18660836830bd126f721f7ccb94c7b/package.json#L55

Reply:
@datho7561 please remove mocha then

Reply:
Sorry, I forgot I hadn't pushed this change yet.